sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,5871 +1,6462 @@
StudyCode: \"18\"\r\n  },\r\n  {\r\n    sabcCode: \"W\",\r\n\tlevelOfStudyCode: \"4\",\r\n\tfederalStudyCode: \"33\"\r\n  },\r\n  {\r\n    sabcCode: \"W\",\r\n\tlevelOfStudyCode: \"2\",\r\n\tfederalStudyCode: \"18\"\r\n  },\r\n  {\r\n    sabcCode: \"W\",\r\n\tlevelOfStudyCode: \"7\",\r\n\tfederalStudyCode: \"18\"\r\n  },\r\n  {\r\n    sabcCode: \"W\",\r\n\tlevelOfStudyCode: \"5\",\r\n\tfederalStudyCode: \"48\"\r\n  },\r\n  {\r\n    sabcCode: \"W\",\r\n\tlevelOfStudyCode: \"1\",\r\n\tfederalStudyCode: \"18\"\r\n  },\r\n  {\r\n    sabcCode: \"W\",\r\n\tlevelOfStudyCode: \"6\",\r\n\tfederalStudyCode: \"63\"\r\n  },\r\n  {\r\n    sabcCode: \"X\",\r\n\tlevelOfStudyCode: \"5\",\r\n\tfederalStudyCode: \"47\"\r\n  },\r\n  {\r\n    sabcCode: \"X\",\r\n\tlevelOfStudyCode: \"2\",\r\n\tfederalStudyCode: \"17\"\r\n  },\r\n  {\r\n    sabcCode: \"X\",\r\n\tlevelOfStudyCode: \"1\",\r\n\tfederalStudyCode: \"17\"\r\n  },\r\n  {\r\n    sabcCode: \"X\",\r\n\tlevelOfStudyCode: \"4\",\r\n\tfederalStudyCode: \"32\"\r\n  },\r\n  {\r\n    sabcCode: \"X\",\r\n\tlevelOfStudyCode: \"8\",\r\n\tfederalStudyCode: \"17\"\r\n  },\r\n  {\r\n    sabcCode: \"X\",\r\n\tlevelOfStudyCode: \"3\",\r\n\tfederalStudyCode: \"32\"\r\n  },\r\n  {\r\n    sabcCode: \"X\",\r\n\tlevelOfStudyCode: \"6\",\r\n\tfederalStudyCode: \"62\"\r\n  },\r\n  {\r\n    sabcCode: \"X\",\r\n\tlevelOfStudyCode: \"7\",\r\n\tfederalStudyCode: \"17\"\r\n  },\r\n  {\r\n    sabcCode: \"Y\",\r\n\tlevelOfStudyCode: \"4\",\r\n\tfederalStudyCode: \"31\"\r\n  },\r\n  {\r\n    sabcCode: \"Y\",\r\n\tlevelOfStudyCode: \"8\",\r\n\tfederalStudyCode: \"16\"\r\n  },\r\n  {\r\n    sabcCode: \"Y\",\r\n\tlevelOfStudyCode: \"5\",\r\n\tfederalStudyCode: \"46\"\r\n  },\r\n  {\r\n    sabcCode: \"Y\",\r\n\tlevelOfStudyCode: \"6\",\r\n\tfederalStudyCode: \"61\"\r\n  },\r\n  {\r\n    sabcCode: \"Y\",\r\n\tlevelOfStudyCode: \"2\",\r\n\tfederalStudyCode: \"16\"\r\n  },\r\n  {\r\n    sabcCode: \"Y\",\r\n\tlevelOfStudyCode: \"3\",\r\n\tfederalStudyCode: \"31\"\r\n  },\r\n  {\r\n    sabcCode: \"Y\",\r\n\tlevelOfStudyCode: \"1\",\r\n\tfederalStudyCode: \"16\"\r\n  },\r\n  {\r\n    sabcCode: \"Y\",\r\n\tlevelOfStudyCode: \"7\",\r\n\tfederalStudyCode: \"16\"\r\n  },\r\n  {\r\n    sabcCode: \"Z\",\r\n\tlevelOfStudyCode: \"7\",\r\n\tfederalStudyCode: \"15\"\r\n  },\r\n  {\r\n    sabcCode: \"Z\",\r\n\tlevelOfStudyCode: \"8\",\r\n\tfederalStudyCode: \"15\"\r\n  },\r\n  {\r\n    sabcCode: \"Z\",\r\n\tlevelOfStudyCode: \"5\",\r\n\tfederalStudyCode: \"45\"\r\n  },\r\n  {\r\n    sabcCode: \"Z\",\r\n\tlevelOfStudyCode: \"4\",\r\n\tfederalStudyCode: \"30\"\r\n  },\r\n  {\r\n    sabcCode: \"Z\",\r\n\tlevelOfStudyCode: \"2\",\r\n\tfederalStudyCode: \"15\"\r\n  },\r\n  {\r\n    sabcCode: \"Z\",\r\n\tlevelOfStudyCode: \"3\",\r\n\tfederalStudyCode: \"30\"\r\n  },\r\n  {\r\n    sabcCode: \"Z\",\r\n\tlevelOfStudyCode: \"1\",\r\n\tfederalStudyCode: \"15\"\r\n  },\r\n  {\r\n    sabcCode: \"Z\",\r\n\tlevelOfStudyCode: \"6\",\r\n\tfederalStudyCode: \"6\"\r\n  }\r\n]",
```

Comment:
As we talked in the meeting, please format these code on VSCode and paste it back here.

Reply:
Updated.